DirectXTex/DirectXTexConvert.cpp

```diff
@@ -4441,6 +4458,12 @@ namespace
             return false;
         }
 
+        if (filter & (TEX_FILTER_FLOAT16_SATURATE_TO_INF | TEX_FILTER_FLOAT16_KEEP_NANS))
```

Comment:
Before I can sign off here, I really need to see what WIC does. Does it preserve specials or always clamp them?